src/python/director/ikplanner.py

```diff
@@ -1478,6 +1504,7 @@ def runIkTraj(self, constraints, poseStart, poseEnd, nominalPoseName='q_nom', ik
         ikParameters = self.mergeWithDefaultIkParameters(ikParameters)
 
         self.lastManipPlan, info = self.plannerPub.processTraj(constraints, ikParameters, positionCosts, nominalPoseName=nominalPoseName, seedPoseName=poseStart, endPoseName=poseEnd)
+        self.lastManipPlan = self.clipPlan(self.lastManipPlan)
```

Comment:
i think that this plan object isn't actually the plan object that it published when a plan is committed.  There is a separate class which receives plan messages (CANDIDATE_MANIP_PLAN) and assigns those plans to the plan playback panel.  When you click "Execute" then the plan is published on COMMITTED_MANIP_PLAN.  For example, think of LCM log playback, plans come from LCM, not from the ikplanner.  This is a confusing aspect and sometimes, we programmatically commit plans from the ikplanner, so this is something we should probably fix.